### PR TITLE
Finally fixes borg OOC prefs (WOOOT)

### LINKED ISSUE #1475: Cyborgs don't get OOC notes
#### Brief description of the issue
Players playing as a cyborg/robot/drone don't get their OOC notes displayed

#### What you expected to happen
Join the game and have my OOC notes displayed under flavor text like they normally do.

#### What actually happened
No OOC notes

#### Steps to reproduce
-Join as the cyborg job
-Have no OOC notes

#### Code Revision
- (Found using the "Show Server Revision" verb under the OOC tab.)
 Server revision: release - 2017-04-30
92420a6da5a611d0f561aa7a4e8785d2e439c50b

### PR DESCRIPTION
**Pull request by Funce**
Fixes #1475 

I DUNNIT, SHOWER ME IN AFFECTION. (Or not because I feel like only I play Borg sometimes)
If a changelog is needed lemme know!
image

### CODE REVIEW

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 1]
```diff
@@ -310,6 +310,10 @@
 			flavor_text = module_flavour
 		else
 			flavor_text = client.prefs.flavour_texts_robot["Default"]
+		// and meta info - VORE EDIT
```
Comment (SpadesNeil) [inline code/modules/mob/living/silicon/robot/robot.dm]: I'd like it to say specifically "Vorestation Edit" so we can easily Ctrl+F it if we need to

File: code/modules/mob/living/silicon/robot/examine.dm [hunk 2]
```diff
@@ -30,6 +30,12 @@
 		if(UNCONSCIOUS)		msg += "<span class='warning'>It doesn't seem to be responding.</span>\n"
 		if(DEAD)			msg += "<span class='deadsay'>It looks completely unsalvageable.</span>\n"
 	msg += attempt_vr(src,"examine_bellies_borg",args) //VOREStation Edit
+
+	// VOREStation Start
```
Comment (SpadesNeil) [inline code/modules/mob/living/silicon/robot/examine.dm]: "Vorestation Edit START"

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 3]
```diff
@@ -765,6 +769,11 @@
 	if (href_list["showalerts"])
 		subsystem_alarm_monitor()
 		return 1
+	// VOREStation Start
```
Comment (SpadesNeil) [inline code/modules/mob/living/silicon/robot/robot.dm]: "Vorestation Edit START"

**Comment by Funce** [comment]
Cheers @SpadesNeil

**Comment by AeroWing** [comment]
You aren't the only one playing borg out there Funce! Thank you for fixing this for us! <3

**Comment by Funce** [comment]
Cheers AeroWing!

**Pull request merged**